framework/web/AssetManager.php

```diff
@@ -72,6 +72,14 @@ class AssetManager extends Component
      */
     public $baseUrl = '@web/assets';
     /**
+     * @var boolean|string the URI scheme to use. Possible values:
+     *
+     * - `false` (default): assets URL without host info.
+     * - `true`: assets URL with an absolute base URL whose scheme is the same as that in [[\yii\web\UrlManager::hostInfo]].
+     * - string: assets URL with an absolute base URL with the specified scheme (either `http` or `https`).
+     */
+    public $scheme = false;
```

Comment:
Tag has been added.
> Could be defaultScheme to reflect the purpose better.

It's not a defaultScheme.
> it will not use scheme value from asset manager as default scheme. I've updated description in AssetManager: scheme in AssetManager is for assets from assetMap. Only if resolveAsset() found $actualAsset, AssetManager scheme will be used.